### DISCUSSION TITLE
32bit to 16 bit audio data conversion ..quality reduce ?

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#732] [Q&A] [answered]
encoder need 16 bit data and i2s microphone INMP441  provide 32 bit data , if we convert 32 bit to 16 bit that means high amplitude from 32 bit will chop right ?

 means reduce of audio quality right ?

**Comment by pschatzmann** [comment, marked as answer]
In theory yes, in practice no: You will not be able to hear any difference.
E.g CDs are recorded with 16 bits!
